demo.py

```diff
@@ -1,11 +1,15 @@
-import pandas as pd
+'import pandas as pd
```

Comment:
It looks like this is the reason unit-test is failing - quote in front of the import